.pre-commit-config.yaml

```diff
@@ -79,12 +79,12 @@ repos:
             '--filter=-readability/casting,-runtime/int,-build/include_subdir,-readability/fn_size'
         ]
 -   repo: https://github.com/pycqa/pylint
-    rev: v2.16.2
+    rev: v3.0.0a6
```

Comment:
looks like pylint have pushed a new tag, but no new release. I'd suggest reverting this change for now

Reply:
ping on this